scripts/navigation.js

```diff
@@ -40,10 +41,26 @@ onhashchange = (event) => {
     }
     return;
   }
-  const [ type, name, other ] = event.newURL.replace(/.*#!/, '').split('/').map(i => decodeURI(i || '').replace(/_/g, ' '));
+  let [ type, name, other ] = event.newURL.replace(/.*#!/, '').split('/').map(i => decodeURI(i || '').replace(/_/g, ' '));
   if (type == 'loading') {
     return;
   }
+  const originalType = type;
+  const originalName = name;
+  let redirectTarget;
+  let redirectCount = 0;
+  while (redirectTarget = redirections.redirect({type, name}) && redirectCount < 30) {
+    redirectCount++;
+    type = redirectTarget.type;
+    name = redirectTarget.name;
+    //TODO: remove debug log
+    console.log(`Redirecting from ${originalType}/${originalName} to ${type}/${name} [Redirect ${redirectCount}]`);
```

Comment:
Maybe just change it to a console.debug?
```suggestion
    console.debug(`Redirecting from ${originalType}/${originalName} to ${type}/${name} [Redirect ${redirectCount}]`);
```

Reply:
I don't think I'd be allowed to have *any* console.whatever statements in my code at work so I usually mark them with TODO comments and get rid of them before having someone review it 😅 But I guess in this case it would be okay to have, especially in a WIP project.